src/main/drivers/io.c

```diff
@@ -238,19 +238,25 @@ void IOToggle(IO_t io)
 // claim IO pin, set owner and resources
 void IOInit(IO_t io, resourceOwner_e owner, uint8_t index)
 {
+    if (!io)
```

Comment:
Personally I'd prefer to check in serial_init (all CPUs) and avoid initializing undefined pins.

But even then the check should be here, but maybe just hardfault() ?